python/private/python.bzl

```diff
@@ -288,6 +295,7 @@ def _python_impl(module_ctx):
         loaded_platforms[full_python_version] = python_register_toolchains(
             name = toolchain_info.name,
             _internal_bzlmod_toolchain_call = True,
+            platforms = PLATFORMS | py.config.platform_overrides.get(full_python_version, {}),
```

Comment:
I would prefer using the PLATFORMS global once at the creation of py.config so that we could get rid of the global variable altogether at a later time and we would not need to change the interfaces.

Reply:
I had the same thought, but had trouble figuring out how to wire it that way. Did you have something in mind? Do you just mean putting the global on `py.config` (where "defaults" and "platform_overrides" are assigned, in get_toolchain_config)? And then this line changes to `platforms = py.config.platforms | py.config.platform_overrides[version]` ?

Reply:
Yeah I was thinking about something like that. This is very similar how the `uv` version specific things get.

Reply:
Done